src/main/java/com/marianbastiurea/lifeofbees/Queen.java

```diff
@@ -20,8 +20,34 @@ of her life, the new queen will kill her (called 'supercedure').  If the old que
  */
 
 public class Queen {
+    private int ageOfQueen;
```

Comment:
Pick an initial age for the queen in days.

Reply:
done. 300 days